shared/navLinks/messages.ts

```diff
@@ -61,6 +61,14 @@ export const messages = defineMessages({
     id: "nav.mineIronDescription",
     defaultMessage: "Learn how to mine $IRON.",
   },
+  cliTitle: {
+    id: "nav.cliTitle",
+    defaultMessage: "CLI Node",
```

Comment:
`CLI Node` or `Node CLI`? Just wondering since we have `Node App`

Reply:
I wanted to put CLI since that's the primary thing people will probably be looking for but I'm perfectly fine swapping the words.

Reply:
Changed to `Node CLI`